site/public/css/electronic.css

```diff
@@ -171,13 +171,25 @@ button > i.fas:hover {
     color: initial;
 }
 
+:root {
+    --default-border-color: black;
+    --default-box-shadow: 2px 2px 4px black;
+}
+
+@media (prefers-color-scheme: dark) {
+    :root {
+        --default-border-color: grey;
+        --default-box-shadow: 2px 2px 4px grey;
```

Comment:
This color choice causes the outline to not be visible in the Dark (Black) mode.
![image](https://github.com/user-attachments/assets/166738d4-602e-4e96-9504-b61d2e7a4bd3)

Reply:
![Screenshot 2024-07-19 at 8 55 19 AM](https://github.com/user-attachments/assets/9eae6c60-217d-4510-816a-1ab15b34e09e)

It is working on my end in the Dark (Black) mode. Im currently testing it on the browser's inspection tool. How should I go about to perform more testing?